<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.112, y1:0.107318, x2:1, y2:0.994, stop:0 rgba(202, 216, 225, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>160</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Noto Mono&quot;;
color: rgb(52, 52, 52);</string>
    </property>
    <property name="text">
     <string>4IND</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="enrollBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#enrollBtn{
	border-radius:20px;
	font: 75 12pt &quot;Manjari Bold&quot;;
	background-color: rgba(26, 36, 122, 200);
	color: rgb(250, 250, 250);
}

QPushButton#enrollBtn:hover{
	background-color: rgba(26, 36, 122, 180);

}</string>
    </property>
    <property name="text">
     <string>O K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancleBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#cancleBtn{
	border-radius:20px;
	font: 75 12pt &quot;Manjari Bold&quot;;
	background-color: rgba(191, 210, 225, 200);
	color: rgb(52, 52, 52);
}

QPushButton#cancleBtn:hover{
	background-color: rgba(191, 210, 225, 160);
}
</string>
    </property>
    <property name="text">
     <string>C a n c l e</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
font: 11pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
font: 11pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLabel" name="chLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>301</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Sans Serif&quot;;
color: rgb(212, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
font: 11pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>confirm password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
